api/cloudcontroller/ccv2/service_plan_test.go

```diff
@@ -208,4 +208,93 @@ var _ = Describe("Service Plan", func() {
 			})
 		})
 	})
+
+	Describe("UpdateServicePlan", func() {
+		When("the service plan exists", func() {
+			var (
+				expectedRequestBody map[string]interface{}
+				warnings            Warnings
+				err                 error
```

Comment:
Could you rename `err` to `executeErr`? We've been bitten in the past by accidentally shadowing variables called `err` that are scoped beyond the immediate block.